firstvoices/backend/tests/test_apis/test_category_api.py

```diff
@@ -104,3 +101,155 @@ def test_detail_404(self):
         wrong_endpoint = self.get_detail_endpoint(self.site.slug, "54321")
         response = self.client.get(wrong_endpoint)
         assert response.status_code == 404
+
+    @pytest.mark.django_db
+    def test_contains_word(self):
+        word_entry = DictionaryEntryFactory(site=self.site)
+        category_word = ParentCategoryFactory(site=self.site)
+        category_word.dictionary_entries.add(word_entry)
+
+        # Testing for WORD flag
+        response = self.client.get(
+            self.get_list_endpoint(
+                self.site.slug, query_kwargs={"contains": TypeOfDictionaryEntry.WORD}
+            )
+        )
+        response_data = json.loads(response.content)
+
+        assert response.status_code == 200
+        assert response_data["count"] > 0
+
+        assert response_data["results"][0] == {
+            "id": str(category_word.id),
+            "title": category_word.title,
+            "description": category_word.description,
+            "children": [],
```

Comment:
No need to test the entire data object here, since there are already tests for the data format. This test only needs to check for the correct ids

Reply:
(Same for all of these new tests)